packages/orbit-components/.eslintrc.cjs

```diff
@@ -26,12 +26,13 @@ module.exports = {
         "flowtype/require-valid-file-annotation": ["error", "always"],
       },
     },
-    {
-      files: ["src/**/*.ts?(x)"],
-      rules: {
-        "@typescript-eslint/consistent-type-imports": "error",
-      },
-    },
+    // TODO: temporary disabled because of the issue: https://github.com/typescript-eslint/typescript-eslint/issues/4608
+    // {
+    //   files: ["src/**/*.ts?(x)"],
+    //   rules: {
+    //     "@typescript-eslint/consistent-type-imports": "warn",
```

Comment:
It's unfortunate that we lose this rule tbh. Just to be clear, this became buggy after the update? And was never fixed?

Reply:
As far as I know it's still not fixed, I will check more on that 🤔